action.yml

```diff
@@ -12,6 +12,14 @@ inputs:
   lambda_function_name:
     description: The Lambda function name. Check the AWS docs/readme for examples.
     required: true
+  use_s3:
+    description: Whether to use S3 (true) or not (false) for the dependencies layer upload.
+    required: false
+    default: 'false'
+  s3_bucket_name:
+    description: The S3 bucket name for the dependencies layer upload.
+    required: false
+    default: no-bucket-name-here
```

Comment:
`s3_bucket_name` is then conditionally required, right? If `use_s3` is true `s3_bucket_name` must be set. If the user does not set it, it will try to upload to `no-bucket-name-here`. I don't think this should be supported; the error messages might not make sense to the user, and there is a chance, however tiny, that someone owns this bucket. 

**Possible solutions:**
- Remove `use_s3` and only keep `s3_bucket_name` which would not have a default and if set, we upload to S3
- Add a conditional in the bash script to not upload to S3 if the `s3_bucket_name` is not explicitly set

Either way I don't think we should have a default value for this. What do you think?